internal/dao/tests/noop_job_test.go

```diff
@@ -107,92 +107,6 @@ func TestReservationNoopOneSuccess(t *testing.T) {
 				require.Equal(t, "No operation finished", updatedRes.Status)
 			},
 		},
-		{
-			name: "ThreeStepsNoError",
-			action: func() *models.NoopReservation {
-				// create new reservation
-				res := &models.NoopReservation{
-					Reservation: models.Reservation{
-						AccountID:  1,
-						Steps:      3,
-						StepTitles: []string{"Step one", "Step two", "Step three"},
-						Provider:   models.ProviderTypeNoop,
-						Status:     "Created",
-					},
-				}
-				err := reservationDao.CreateNoop(ctx, res)
-				require.NoError(t, err)
-				require.NotZero(t, res.ID)
-
-				// create new job
-				successJob := dejq.PendingJob{
-					Type: queue.TypeNoop,
-					Body: &jobs.NoopJobArgs{
-						AccountID:     1,
-						ReservationID: res.ID,
-						Fail:          false,
-					},
-				}
-				_, err = queue.GetEnqueuer().Enqueue(context.Background(), successJob, successJob, successJob)
-				require.NoError(t, err)
-
-				return res
-			},
-			checks: func(updatedRes *models.Reservation) {
-				require.True(t, updatedRes.Success.Bool)
-				require.Empty(t, updatedRes.Error)
-				require.Equal(t, int32(3), updatedRes.Step)
-				require.Equal(t, int32(3), updatedRes.Steps)
-				require.Equal(t, "No operation finished", updatedRes.Status)
-			},
-		},
-		{
-			name: "ThreeStepsFirstFails",
-			action: func() *models.NoopReservation {
-				// create new reservation
-				res := &models.NoopReservation{
-					Reservation: models.Reservation{
-						AccountID:  1,
-						Steps:      3,
-						StepTitles: []string{"Step one", "Step two", "Step three"},
-						Provider:   models.ProviderTypeNoop,
-						Status:     "Created",
-					},
-				}
-				err := reservationDao.CreateNoop(ctx, res)
-				require.NoError(t, err)
-				require.NotZero(t, res.ID)
-
-				// create new job
-				successJob := dejq.PendingJob{
-					Type: queue.TypeNoop,
-					Body: &jobs.NoopJobArgs{
-						AccountID:     1,
-						ReservationID: res.ID,
-						Fail:          false,
-					},
-				}
-				failureJob := dejq.PendingJob{
-					Type: queue.TypeNoop,
-					Body: &jobs.NoopJobArgs{
-						AccountID:     1,
-						ReservationID: res.ID,
-						Fail:          true,
-					},
-				}
-				_, err = queue.GetEnqueuer().Enqueue(context.Background(), failureJob, successJob, successJob)
-				require.NoError(t, err)
-
-				return res
-			},
-			checks: func(updatedRes *models.Reservation) {
-				require.False(t, updatedRes.Success.Bool)
-				require.Equal(t, "step skipped: step skipped: job failed on request", updatedRes.Error)
-				require.Equal(t, int32(3), updatedRes.Step)
-				require.Equal(t, int32(3), updatedRes.Steps)
-				require.Equal(t, "No operation finished", updatedRes.Status)
-			},
-		},
```

Comment:
The final version does not support steps (dependant jobs), so these tests are irrelevant now.